### DISCUSSION TITLE
Open Source Actuarial Experience

### DISCUSSION THREAD

**Discussion by jkoestner** [actuarialopensource/website#43] [Show and tell]
I'm a big fan of the open source and appreciate one of the tools this group has created - `pymort`.

I've been creating a `python` implementation of reviewing mortality and trying to incorporate machine learning techniques into data analytics.  This isn't yet in a "release" state and currently adding in credibility to the mix. It'll be in "release" state in 2 months. This project has mainly been an exercise of learning.

https://github.com/jkoestner/morai

**Comment by MatthewCaseres** [comment]
Hey thanks for this, glad someone has found the project useful/interesting. Your `morai` project shows off many aspects of software development, like publishing docker containers from CI, GitHub actions in general, testing, docstrings, etc. Love to see actuaries adopting these tools.

You are resolving an open issue it seems - https://github.com/actuarialopensource/pymort/issues/16. Once you have officially released your project I will close out that issue and just reference your project.

One thing related to this docker publishing in CI you are doing is some benchmarks we are running. Work is underway to rework certain life insurance modeling benchmarks so that they run on GPU and are distributed as containers. I was thinking the other day to just publish them outside of CI to get it done hastily, your CI pipeline is a nice reference if I decide to automate this.

**Reply by jkoestner** [reply]
Nice. Yeah I'll respond back to this when it's updated. The more I look into GitHub the more that I find. This repo is helpful putting some of it in already to find place.


It was interesting to see such a mix of Julia/R/Python.

**Reply by jkoestner** [reply]
Is there anyway to get more involved in this project? I also have some material for credibilty.

**Reply by MatthewCaseres** [reply]
There is no formal structure but eveything is open to pull requests and issues. Credibility sounds like it might fit in with some of Matt Heaphy's python work (experience study related)?

Right now this org is mostly dormant. But I try to keep up with a couple of projects I'm familiar with (JuliaActuary, Lewis Fogden's packages, lifelib) and contribute when I see an opportunity.
